conda-store-server/tests/test_auth.py

```diff
@@ -140,36 +141,191 @@ def test_authorization(conda_store, entity_bindings, arn, permissions, authorize
     assert authorized == authorization.authorize(entity, arn, permissions)
 
 
-def test_end_to_end_auth_flow(conda_store):
+@pytest.mark.parametrize("role_mappings_version", [1, 2])
```

Comment:
Updated this test to match the newly added `test_end_to_end_auth_flow_v2`. The v1 test calls HTTP APIs that use the v1 role mappings table. The v2 test does the same for the v2 table.